src/core/aws-ses.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@nestjs/common';
+import * as AWS from 'aws-sdk';
+import { EnvironmentService } from './config/environment.service';
+
+@Injectable()
+export class SesService {
```

Comment:
`SES` can be made injectable with a factory so there's no need for a wrapper service.
Take a look at [`aws-s3.factory.ts`](https://github.com/SeedCompany/cord-api-v3/blob/aba8071027962ea21df4b6a7e97d845e2c2d4f8d/src/core/aws-s3.factory.ts)
This can be referenced in the `providers` & `exports` of `CoreModule` allowing `SES` to be injected into any service.